opentelemetry-sdk/src/metrics/mod.rs

```diff
@@ -2412,6 +2414,123 @@ mod tests {
             empty_attrs_data_point.value, 6,
             "Empty attributes value should be 3+3=6"
         );
+
+        // Phase 2 - for delta temporality, after each collect, data points are cleared
+        if temporality == Temporality::Delta {
```

Comment:
You can also check that these measurements continue to go to overflow for Cumulative temporality.